circle.yml

```diff
@@ -10,6 +10,11 @@ test:
   pre:
     - mozdownload --version latest-esr --destination firefox.tar.bz2
     - mozinstall firefox.tar.bz2
+    - wget "https://github.com/mozilla/geckodriver/releases/download/v0.9.0/geckodriver-v0.9.0-linux64.tar.gz"
+    - gunzip -c geckodriver-v0.9.0-linux64.tar.gz | tar xopf -
+    - mv geckodriver wires
+    - chmod +x wires
```

Comment:
Let's `chmod +x geckodriver` and then move it to `/home/ubuntu/bin/wires` to save us the additional `mv`.